CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.1]
+
+### Fixed
+
+  - Resolved an issue where livestream audio would continue playing even after moving away from the tab.
```

Comment:
do we usually give this much detail for supporting libs? Or do I just mention bug fix?